src/common/common.c

```diff
@@ -0,0 +1,30 @@
+#include<oqs/common.h>
+
+#include<string.h>
+
+#if defined(WINDOWS)
+#include<windows.h>
+#endif
+
+void OQS_MEM_cleanse(void *ptr, size_t len) {
+#if defined(WINDOWS)
+	SecureZeroMemory(pnt, len);
+#elif defined(HAVE_MEMSET_S)
+	memset_s(pnt, (rsize_t) len, 0, (rsize_t) len);
```

Comment:
This line should be deleted, you're doing a memset_s on the next line in the if condition